examples/swapi/src/AddPerson.re

```diff
@@ -0,0 +1,37 @@
+let ste = ReasonReact.stringToElement;
+
+module AddPerson = [%graphql {|
+    mutation addPerson($age: Int!, $name: String!) {
+        createPerson(age: $age, name: $name) {
+          name
+        }
+      }
+|}];
+
+let component = ReasonReact.statelessComponent("AddPerson");
+
+module AddPersonMutation = ReasonApollo.CreateMutation(AddPerson);
+
+let make = (_children) => {
+    ...component,
+    render: _self => {
+        let dic = Js.Dict.empty();
+        Js.Dict.set(dic, "age", Js.Json.number(24 |> float_of_int));
+        Js.Dict.set(dic, "name", Js.Json.string("Bob"));
+        let variables = Js.Json.object_(dic);
```

Comment:
I think it's better to show how to pass the variables using the `make` from graphql_ppx for more safety

Reply:
Absolutely!

Reply:
Pushed